framework/wcf/Extensibility/Transport/AzureQueueStorageClient/AzureQueueStorageClient/AzureQueueStorageChannelFactory.cs

```diff
@@ -0,0 +1,117 @@
+﻿//----------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+//----------------------------------------------------------------
+
+using System;
+using System.Collections.ObjectModel;
+using System.ServiceModel;
+using System.ServiceModel.Channels;
+
+namespace Microsoft.Samples.AzureQueueStorage
+{
+    /// <summary>
+    /// IChannelFactory implementation for AzureQueueStorage.
+    /// </summary>
+    class AzureQueueStorageChannelFactory : ChannelFactoryBase<IOutputChannel>
+    {
```

Comment:
You should make sure everything has an accessibility modifier. For example, this is internal. There's a VS code analyzer which will suggest this and can apply a quick fix to an entire project at once.

Reply:
Thanks for the suggestion. fixed across entire project.